src/test/java/de/tum/in/www1/artemis/service/plagiarism/ContinuousPlagiarismControlServiceTest.java

```diff
@@ -0,0 +1,69 @@
+package de.tum.in.www1.artemis.service.plagiarism;
+
+import static org.mockito.Mockito.*;
+
+import java.io.IOException;
+import java.util.Set;
+
+import org.junit.jupiter.api.Test;
+
+import de.jplag.exceptions.ExitException;
+import de.tum.in.www1.artemis.domain.Exercise;
+import de.tum.in.www1.artemis.domain.FileUploadExercise;
+import de.tum.in.www1.artemis.domain.ProgrammingExercise;
+import de.tum.in.www1.artemis.domain.TextExercise;
+import de.tum.in.www1.artemis.domain.modeling.ModelingExercise;
+import de.tum.in.www1.artemis.domain.quiz.QuizExercise;
+import de.tum.in.www1.artemis.repository.ExerciseRepository;
+
+class ContinuousPlagiarismControlServiceTest {
```

Comment:
Is there a reason why you test completely outside of our framework?

Reply:
It uses Junit as any other test class. This is unit test, so `@SpringBootTest` is not needed